.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ missingdeps: &missingdeps
 
 test: &test
   name: Test bundle
-  command: bundle exec rake test
+  command: bundle exec rake --trace
```

Comment:
Why this change?

Reply:
it makes it run rubocop and the test
added --trace so it's obvious if test failed or rubocop, it will show the task name